src/transformers/data/datasets/squad.py

```diff
@@ -128,28 +129,35 @@ def __init__(
         with FileLock(lock_path):
             if os.path.exists(cached_features_file) and not args.overwrite_cache:
                 start = time.time()
-                self.features = torch.load(cached_features_file)
+                cache_dict = torch.load(cached_features_file)
```

Comment:
Should still keep this as `self.old_features` just in case.

Reply:
Fixed